samples/Mvc.DynamicCorsPolicyProvider/DynamicCorsPolicyProvider.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+
+namespace Mvc.DynamicCorsPolicyProvider
+{
+    public class DynamicCorsPolicyProvider : ICorsPolicyProvider
+    {
+        private readonly PhysicalFileProvider _fileProvider;
+        private readonly ILogger<DynamicCorsPolicyProvider> _logger;
+        private Dictionary<string, CorsPolicy> _policies = null;
+
+        public DynamicCorsPolicyProvider(PhysicalFileProvider fileProvider, ILogger<DynamicCorsPolicyProvider> logger)
```

Comment:
Why does this depend on `PhysicalFileProvider` specifically, and not just the interface?